apps/cowswap-frontend/src/modules/application/containers/App/menuConsts.tsx

```diff
@@ -5,6 +5,7 @@ import AppziButton from 'legacy/components/AppziButton'
 import { FortuneWidget } from 'modules/fortune/containers/FortuneWidget'
 
 import { Routes } from 'common/constants/routes'
+import { Version } from '../../../../legacy/components/Version'
```

Comment:
Should we move this from legacy to a proper folder elsewhere? As it's not legacy more?

Reply:
Maybe. But you can shorten the import a bit:

```suggestion
import { Version } from 'legacy/components/Version'
```

Reply:
Thanks 👍🏼